src/webserver.js

```diff
@@ -194,12 +193,13 @@ function setupExpressApp(app, callback) {
 	app.use(middleware.addHeaders);
 	app.use(middleware.processRender);
 	auth.initialize(app, middleware);
+	app.use(middleware.autoLocale);	// must be added after auth middlewares are added
 
 	var toobusy = require('toobusy-js');
 	toobusy.maxLag(meta.config.eventLoopLagThreshold);
 	toobusy.interval(meta.config.eventLoopInterval);
 
-	setupAutoLocale(app, callback);
+	callback();
```

Comment:
Don't anticipate this would cause issues. Was initially waiting for autolocale logic to run because the languages list needs to be populated. Only consideration is that the actual `app.use` is called after the auth middlewares are added.